src/Interactive/HostProcess/InteractiveHostEntryPoint.cs

```diff
@@ -15,6 +16,7 @@ namespace Microsoft.CodeAnalysis.Interactive
 {
     internal static class InteractiveHostEntryPoint
     {
+        [SupportedOSPlatform("windows")]
```

Comment:
Is there not a constant for these?

Reply:
I didn't see one. dotnet/runtime seems to use strings.